.storybook/stories/DataFiltering/components/DataList.tsx

```diff
@@ -22,36 +22,39 @@ type DataListProps = {
 export const DataList = ({ loading, data }: DataListProps) => {
 	const { resetFilters } = useSortAndFilterContext();
 
-	if (loading) {
-		return (
-			<Stack as="ul" gap={1} id={tableId}>
-				{Array.from({ length: 10 }, (_, i) => i).map((i) => (
-					<DataListItemSkeleton key={i} />
-				))}
-			</Stack>
-		);
-	}
-
-	if (!data.length)
-		return (
-			<Stack gap={2} alignItems="flex-start" id={tableId} role="status">
-				<Stack gap={1}>
-					<HelpIcon size="lg" color="muted" />
-					<Heading type="h2" fontSize="lg">
-						No results found
-					</Heading>
-					<Text>Try adjusting your filter options.</Text>
+	return (
+		<div id={tableId}>
+			{loading ? (
+				<Stack as="ul" gap={1}>
+					{Array.from({ length: 10 }, (_, i) => i).map((i) => (
```

Comment:
Mightn't be an issue for us right now but this should probably reflect the "items per page" value, instead of being hardcoded to 10.

Reply:
Good pickup. Wanna fix now?

Reply:
Yeah, may as well. [This is what we did in yourgov](https://github.com/agriculturegovau/agds-next/blob/develop/yourgov/components/Staff/DataTable.tsx#L164).

Reply:
Do you want me to add that in?

Reply:
Go for it